web-client/src/components/PopUpForgotPassword.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Dialog from "@material-ui/core/Dialog";
+
+const styles = makeStyles({
+  main: {
+    display: "inline-block",
+  },
+  content: {
+    padding: "30px 100px",
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    float: "left",
+    textAlign: "left",
+    maxWidth: "400px",
+    margin: "16px",
+  },
+  textField: {
+    marginBottom: "32px",
+    width: "300px",
+  },
+  send: {
+    width: "100px",
+    textTransform: "none",
+    float: "right",
+    marginLeft: "auto",
+    display: "block",
+  },
+  textButton: {
+    textTransform: "none",
+  },
+  message: {
+    textAlign: "center",
+    marginTop: "30px",
+    marginBottom: "30px",
+  },
+});
+
+const sendSuccessfullyStyle = {
+  visibility: "visible",
+  color: "green",
+};
+const sendUnsuccessfullyStyle = {
+  visibility: "visible",
+  color: "red",
+};
+const defaultStyle = {
+  visibility: "hidden",
+};
+
+export default function PopUpForgotPassword({SET_EMAIL}) {
+  const [values, setValues] = React.useState({
+    email: "",
+  });
+
+  const schema = yup.object().shape({
+    email: yup.string().email("Email should have correct format"),
+    //   .required("Required field"),
```

Comment:
Why it's commented?

Reply:
I don't know, if email is required.

Reply:
@upsu90 What do you think?

Reply:
Imo email should be required

Reply:
we can only sign in using email, can't we?

Reply:
It's not login form, but popup to reset password
It should be required, otherwise our app will try to send to empty email